go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/alecthomas/jsonschema v0.0.0-20200530073317-71f438968921
 	github.com/benbjohnson/clock v1.3.5
-	github.com/celestiaorg/celestia-app v1.0.0-rc8
+	github.com/celestiaorg/celestia-app v1.0.0-rc8.0.20230711195230-7a176b2bb786 // v1.0.0-rc8
```

Comment:
we can bump to an official tag after https://github.com/celestiaorg/celestia-app/pull/2081

Reply:
Created https://github.com/celestiaorg/celestia-app/releases/tag/v1.0.0-rc9 I just can't push to this branch b/c permissions issue 😢 

```
~/git/rootulp/celestia/celestia-node on evan/bump-ce..p-v1.0.0-rc9 >1

$ git push
ERROR: Permission to celestiaorg/celestia-node.git denied to rootulp.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
Resolved via https://github.com/celestiaorg/celestia-node/pull/2458/commits/d04618112569487d4437819af08254a6719676fb